src/VoteButton.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+import styled from 'styled-components';
+
+const StyledButton = styled.button`
+  display: inline-flex;
+  border: none;
+  border-radius: 4px;
+  background-color: #bf80ff;
+  color: white;
+  font-weight: bold;
+  cursor: pointer;
+  margin-left: 2rem;
+  padding-left: 1rem;
+  padding-right: 1rem;
+`;
+
+function VoteButton(props) {
+  const [count, setCount] = useState(null);
+  const CastVote = async () => {
+    const id = props.vote_id;
+    try {
+      const data = await axios.get(
+        `http://ec2-13-209-5-166.ap-northeast-2.compute.amazonaws.com:8000/api/vote?id=${id}`
+      );
```

Comment:
```suggestion
      const data = await axios.get(
        `http://ec2-13-209-5-166.ap-northeast-2.compute.amazonaws.com:8000/api/vote`,
        { params = { id }
      );
```
get 두번째 인자로 config를 넘길 수 있습니다.
HTTP 헤더나 Query 문 같은 걸 줄 수 있어요.